src/pages/booklog/BooklogDetailPage.tsx

```
@@ -39,84 +78,139 @@ function TagPill({ children }: { children: React.ReactNode }) {
+          const recPosts = await getBooklogRecommendPosts(Number(postId));
+          if (!alive) return;
+          setRecommendPosts(recPosts);
+        } catch (e) {
+          console.error("추천 인기글 조회 실패", e);
+          if (!alive) return;
+          setRecommendPosts([]);
+        }
+      } catch (e) {
+        console.error("북로그 상세 조회 실패", e);
+      } finally {
+        if (!alive) return;
```

ESLint

[error] Unsafe usage of ReturnStatement.

(no-unsafe-finally)